### ISSUE TITLE
what is s_test and grad_z?

### ISSUE THREAD

**Issue by jiamingzhang94** [nimarb/pytorch_influence_functions#20]
I am confusing with the corresponding relationship of s_test and grad_z in original paper

**Comment by zatcsc** [comment]
As far as I understand, `grad_z` is the gradient of the Loss at a training example, and `s_test` is the product of the Inverse of Hessian and gradient of the Loss at a test example.

**Comment by Hongbo-Z** [comment]
` s_test` is used to calculate the inverse HVP estimation by Stochastic Second-Order Algorithm.
`grad_z` can be used to calculate any test point $\nabla_{\theta}L(z_{\text{test}}, \hat{\theta})$ or any training data point $\nabla_{\theta}L(z, \hat{\theta})$
